ci/travis/build-docker-images.py

```diff
@@ -174,16 +175,17 @@ def _build_docker_image(image_name: str,
                          f" {image_type.keys()}")
 
     # TODO(https://github.com/ray-project/ray/issues/16599):
-    # remove below after supporting ray-ml images with Python 3.9
-    if image_name == "ray-ml" and py_version == "py39":
+    # remove below after supporting ray-ml images with Python 3.9/3.10
+    if image_name == "ray-ml" and (py_version == "py39" or
+                                   py_version == "py310"):
         print(f"{image_name} image is currently unsupported with "
-              "Python 3.9")
+              "Python 3.9/3.10")
         return
 
     build_args = {}
     build_args["PYTHON_VERSION"] = PY_MATRIX[py_version]
-    # I.e. "py36"[-1] == 6
-    build_args["PYTHON_MINOR_VERSION"] = py_version[-1]
+    # I.e. "py36"[3:None] == 6
+    build_args["PYTHON_MINOR_VERSION"] = py_version[3:None]
```

Comment:
is this intentional?

Reply:
Yep, required since the previous logic doesn't work with an input of `py310` (would return 0, instead of the desired 10).